code/game/objects/items/weapons/storage/toolbox.dm

```diff
@@ -5,11 +5,12 @@
 	icon_state = "red"
 	item_state = "toolbox_red"
 	flags = CONDUCT
-	force = 10
-	throwforce = 10
+	force = 15
+	throwforce = 15
 	throw_speed = 2
 	throw_range = 7
 	w_class = 4
+	max_combined_w_class = 28

```

Comment:
this determines how much it can hold, theres another var that determines what size.
BTW, toolboxes can only hold size 2 or smaller. Which is pockets.